src/pages/workspace/WorkspaceInitialPage.tsx

```diff
@@ -437,6 +456,18 @@ function WorkspaceInitialPage({policyDraft, policy: policyProp, route}: Workspac
                         </View>
                     </OfflineWithFeedback>
                 </ScrollView>
+                <View style={[styles.pb4, styles.mh3, styles.mt3]}>
+                    <Text style={[styles.textSupporting, styles.fontSizeLabel, styles.ph4]}>{translate('workspace.common.expense')}</Text>
+                    <MenuItem
```

Comment:
Would it be better if use `OptionRowLHN` instead?

Reply:
I don't think it should be used `OptionRowLHN` because it shows many other things.